pkg/services/secrets/kvstore/migrations/migrator.go

```diff
@@ -25,16 +26,20 @@ type SecretMigrationServiceImpl struct {
 func ProvideSecretMigrationService(
 	serverLockService *serverlock.ServerLockService,
 	dataSourceSecretMigrationService *datasources.DataSourceSecretMigrationService,
+	pluginSecretMigrationService *kvstore.PluginSecretMigrationService,
 ) *SecretMigrationServiceImpl {
 	return &SecretMigrationServiceImpl{
 		ServerLockService: serverLockService,
 		Services: []SecretMigrationService{
 			dataSourceSecretMigrationService,
+			// migration from unified secrets to plugin secrets, if enabled.
+			// pluginMigrationService should always be the last one
+			pluginSecretMigrationService,
 		},
 	}
 }
 
-// Run migration services. This will block until all services have exited.
+// Migrate Run migration services. This will block until all services have exited.
```

Comment:
was this an accident?

Reply:
The IDE suggested it, and using the name of the func, package, struct etc, at the beginning of a comment, means that you are documenting it. 

> Comments that appear before top-level declarations, with no intervening newlines, are considered to document the declaration itself. These “doc comments” are the primary documentation for a given Go package or command. For more about doc comments, see “[Go Doc Comments](https://go.dev/doc/comment)”.

https://go.dev/doc/effective_go#commentary